doc/ref_model/chapters/chapter09.rst

```diff
@@ -113,37 +126,116 @@ There are the following requirement types:
+ -  Human related Risks Mitigation: automation reduces risks related to human errors, rogue activities, and safeguards the institutional knowledge 
+    from leakage in case any employee leaves the organization
+ -  Higher Efficiency: achieved by minimizing human inaccuracies and eliminating the lack of knowledge about infrastructure installed base and its configuration, using
+ the CI/CD techniques adapted to infrastructure 
+ -  Cost/time Saving: engineers save up on time and cost which can be wisely invested in performing higher-value jobs; additional cost savings on cloud more optimal 
+ use of cloud resources using LCM Automation
+ 
+Infrastructure LCM Automation Framework
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+ 
+The following diagrams provide mapping between different stages of the lifecycle automation across all layers of the stack, to owners of infrastructure and cloud and 
+the tenant as the consumer of the cloud services, in three very different scenarios: applications running as containers within virtual machines (CaaS on IaaS scenario),
+application running as containers on bare metal (CaaS on BM scenario) and a more traditional view of applications running as VNFs within virtual machines (IaaS 
+scenario). The diagrams define the scope of the Infrastructure LCM Automation for each of these scenarios. The dotted lines symbolise the interactions between the layers of 
+each of the model.
+ 
+.. figure:: ../figures/RM-Ch09-LCM-Automation-CaaS-on-IaaS.png
+   :name: Infrastructure Automation in CaaS on IaaS scenario
+   :alt: "Infrastructure Automation in CaaS on IaaS scenario"
+
+   Infrastructure Automation in CaaS on IaaS scenario
+   
+In the CaaS on IaaS scenario, the Infrastructure Automation scope covers the Site/Physical layer,  IaaS layer and CaaS layer. From the lifecycle 
+perspective (the left hand side of the diagram), Site/Physical layer is entirely owned by the Infrastructure Owner, the virtualised infrastructure 
+layer (IaaS) is shared between the Infrastructure Owner and the Cloud Provider. Similarly,  the container orchestration layer (CaaS) is shared between 
+the Cloud Provider and the Cloud Consumer / Tenant.   These relationships can be illustrated by a situation, where a telecom operator owns the physical 
+infrastructure on which an external cloud provider runs the virtualisation software (hypervisor).  Sharing CaaS layer between the Cloud Provider and 
+the Cloud Consumer reflects the fact that the container management/orchestration software like Kubernetes is lifecycled by the Cloud Provider 
+(for instance when scaling out containers) but also by the Cloud Consumer because of the very close lifecycle relationship between an application and 
+a container in this model. For instance, destroying an application means also destroying related containers, Hence CaaS can be also considered as a part 
+of the Application Orchestration layer. 
```

Comment:
This seems a bit confusing or I don't understand the sentence. 
Application orchestration should be independent. Scaling out containers should be done by Kubernetes. Destroying an application shouldn't affect the CasS or IaaS platform.

Reply:
@ulikleber, when a Cloud Consumer destroys application, then Kubernetes would destroy containers related to this application.  This sentence simply explains that  the container as as service layer is (form operational perspective) shared between Cloud Consumer and Cloud Provider

Reply:
@ulikleber + @walterkozlowski I think that what is deemed to "application orchestration" or "infrastructure automation" is solely a function of the role of the user/operator and not the tool. So (1) if Cloud Provider is exposing IaaS then responsibility for CaaS devolves to the Tenant and then the CaaS layer is part of "application" or (2) if Cloud Provider is exposing CaaS to tenant then orchestrating just the CNFs on top of this. The purpose is to illustrate these options, but agree that it it critical that what ever is exposed Cloud Provider should not be able to be "disrupted" by the Tenant, which is the "independence" that must be maintained.
NOTE: no change to content for this review.

Reply:
I still find the sentence confusing, but I will not object.